migrations/dlu/7_isOnline.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE accounts ADD if not exists is_online tinyint(1) NOT NULL DEFAULT 0;
```

Comment:
While this does compile and run with MariaDB, all our queries are MySQL compatible and this query is not compatible with MySQL.

A fix would be to remove `if not exists` from the query and just allow an error to show up when the migration is run since all migrations are idempotent and an error is fine when someone attempts to run add column migrations twice.

Reply:
There is also a `BOOLEAN` type in MySQL that should be used as this only has 2 states.

Reply:
> There is also a BOOLEAN type in MySQL that should be used as this only has 2 states.

`BOOLEAN` is just a synonym for `TINYINT` in MySQL, and are stored the same. 

<br>

> A fix would be to remove if not exists from the query and just allow an error to show up when the migration is run since all migrations are idempotent and an error is fine when someone attempts to run add column migrations twice.

Updated